akka-typed/src/main/scala/akka/typed/scaladsl/TimerScheduler.scala

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.typed.scaladsl
+
+import scala.concurrent.duration.FiniteDuration
+
+/**
+ * Support for scheduled `self` messages in an actor.
+ * It is used with `Actor.withTimer`, which also takes care of the
+ * lifecycle of the timers such as cancelling them when the actor
+ * is restarted or stopped.
+ */
```

Comment:
Rephrase to make it less hidden that on a restart a timer must be scheduled again?

```scala
/**
 * Support for scheduled `self` messages in an actor.
 * It is used with `Actor.withTimer`. 
 * Timers are bound to the lifecycle of the actor that owns it, 
 * and thus are cancelled automatically when it is restarted or stopped.
 */
```

?